std/container/array.d

```diff
@@ -977,9 +981,9 @@ $(D r)
     {
         import std.algorithm.mutation : copy;
 
-        enforce(r._outer._data is _data);
-        enforce(_data.refCountedStore.isInitialized);
-        enforce(r._a <= r._b && r._b <= length);
+        assert(r._outer._data is _data);
+        assert(_data.refCountedStore.isInitialized);
+        assert(r._a <= r._b && r._b <= length);
```

Comment:
These three do not belong with this PR, they're a (breaking) change in philosophy.